packages/atomic/src/components/search/result-template-components/atomic-quickview-modal/atomic-quickview-modal.tsx

```diff
@@ -84,22 +90,25 @@ export class AtomicQuickviewModal implements InitializableComponent {
     this.iframeRef = undefined;
     this.content = undefined;
     this.result = undefined;
+    this.interactiveResult = undefined;
   }
 
   private renderHeader() {
     let headerContent: VNode | null = null;
     if (this.result) {
-      const interactiveResult = buildInteractiveResult(this.bindings.engine, {
+      this.interactiveResult = buildInteractiveResult(this.bindings.engine, {
```

Comment:
Issue found while writing the tests: analytics on the header link needs to be a component instance, or you end up with improper reference to the current result.